pkgs/development/python-modules/mastodonpy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, fetchPypi
+, decorator
+, six
+, requests
+, pytz
+, dateutil
+, python_magic
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "Mastodon.py";
+  version = "1.4.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1229wbckanp08bbqajsn31ggvay271d883nca28wpvsaqdsv8ci5";
+  };
+
+  propagatedBuildInputs = [ decorator six requests pytz dateutil python_magic ];
```

Comment:
I guess setuptools is missing here, because I'm getting this error:
```
[nix-shell:~/.cache/nixpkgs-review/pr-65187]$ tootstream 
Traceback (most recent call last):
  File "/nix/store/k2b74bln8j3hkmbqnaws3wdkv6dw4m3m-tootstream-0.3.7/bin/.tootstream-wrapped", line 6, in <module>
    from tootstream.toot import main
  File "/nix/store/k2b74bln8j3hkmbqnaws3wdkv6dw4m3m-tootstream-0.3.7/lib/python3.7/site-packages/tootstream/toot.py", line 22, in <module>
    import pkg_resources  # part of setuptools
ModuleNotFoundError: No module named 'pkg_resources'
```